provider/cmd/pulumi-resource-command/schema.json

```diff
@@ -180,6 +182,47 @@
             "requiredInputs": [
                 "connection"
             ]
+        },
+        "command:remote:CopyFile": {
+            "description": "Copy a local file to a remote host.",
+            "inputProperties": {
+                "connection": {
+                    "description": "The parameters with which to connect to the remote host.",
+                    "$ref": "#/types/command:remote:Connection"
+                },
+                "localPath": {
+                    "description": "The path of the file to be copied.",
```

Comment:
Can/should we support copying a folder (recursively)?

Reply:
That is a non-breaking change, so my thought was we would add it in a later PR.

Reply:
Yep - makes sense - perhaps open an issue to track this future addition.

Reply:
https://github.com/pulumi/pulumi-command/issues/23